get_query_generators.php

```diff
@@ -0,0 +1,336 @@
+<?php
+/**
+ * This file contains functions that returns sql queries for GET requests
+ * according to the API documentation.
+ */
+
+/**
+ * Returns sql query that returns an unordered list of questions.
+ *
+ * @return String A sql command that returns an unorderedlist of questions.
+ */
+function gen_query_questions()
+{
+    return "SELECT question_id, answer_type, content FROM questions;";
+}
+
+/**
+ * Returns sql query that returns an unordered list of sections related to a ta.
```

Comment:
this function can be extended to show all sections a instructor is registered for as well.